packages/multi-select-combo-box/src/vaadin-multi-select-combo-box.js

```diff
@@ -119,65 +121,22 @@ class MultiSelectComboBox extends MultiSelectComboBoxMixin(
           <span part="required-indicator" aria-hidden="true" @click="${this.focus}"></span>
         </div>
 
-        <vaadin-multi-select-combo-box-internal
-          id="comboBox"
-          .owner="${this}"
-          .filteredItems="${this.filteredItems}"
-          .items="${this.items}"
-          .itemIdPath="${this.itemIdPath}"
-          .itemLabelPath="${this.itemLabelPath}"
-          .itemValuePath="${this.itemValuePath}"
-          .disabled="${this.disabled}"
+        <vaadin-multi-select-combo-box-container
+          part="input-field"
+          .autoExpandVertically="${this.autoExpandVertically}"
           .readonly="${this.readonly}"
-          .autoOpenDisabled="${this.autoOpenDisabled}"
-          .allowCustomValue="${this.allowCustomValue}"
-          .overlayClass="${this.overlayClass}"
-          .dataProvider="${this.dataProvider}"
-          .filter="${this.filter}"
-          .lastFilter="${this._lastFilter}"
-          .loading="${this.loading}"
-          .size="${this.size}"
-          .selectedItems="${this.selectedItems}"
-          .selectedItemsOnTop="${this.selectedItemsOnTop}"
-          .itemClassNameGenerator="${this.itemClassNameGenerator}"
-          .topGroup="${this._topGroup}"
-          .opened="${this.opened}"
-          .renderer="${this.renderer}"
-          .keepFilter="${this.keepFilter}"
+          .disabled="${this.disabled}"
+          .invalid="${this.invalid}"
           theme="${ifDefined(this._theme)}"
-          @combo-box-item-selected="${this._onComboBoxItemSelected}"
-          @change="${this._onComboBoxChange}"
-          @custom-value-set="${this._onCustomValueSet}"
-          @filtered-items-changed="${this._onFilteredItemsChanged}"
-          @filter-changed="${this._onComboBoxFilterChanged}"
-          @last-filter-changed="${this._onComboBoxLastFilterChanged}"
-          @loading-changed="${this._onComboBoxLoadingChanged}"
-          @opened-changed="${this._onComboBoxOpenedChanged}"
-          @size-changed="${this._onComboBoxSizeChanged}"
         >
-          <vaadin-multi-select-combo-box-container
-            part="input-field"
-            .autoExpandVertically="${this.autoExpandVertically}"
-            .readonly="${this.readonly}"
-            .disabled="${this.disabled}"
-            .invalid="${this.invalid}"
-            theme="${ifDefined(this._theme)}"
-          >
-            <slot name="overflow" slot="prefix"></slot>
-            <div id="chips" part="chips" slot="prefix">
-              <slot name="chip"></slot>
-            </div>
-            <slot name="input"></slot>
-            <div
-              id="clearButton"
-              part="clear-button"
-              slot="suffix"
-              @touchend="${this._onClearButtonTouchend}"
```

Comment:
This was added in https://github.com/vaadin/web-components/pull/4141. It should't be needed anymore as `_onClearAction()` will call `clear()` and that method will be used from the combo-box base mixin. I will keep the test checking that items are cleared on touchend though.